showcase-app/test-cases.md

```diff
@@ -95,3 +95,31 @@ If you want to run the tests interactively, you use `npm run test:ui`.
   - Reload the page.
   - Verify that the active profile is still displayed.
 
+### 11. Login without extension using WebID
+
+- **Description**: Test the login flow using a WebID input field.
+- **Test Steps**:
+  - Register a profile.
+  - Load main page.
+  - Enter the profile's WebID in the input field, just like a user would.
+  - Log in using profile's credentials.
+  - Verify that upon redirect to the main page, the app is authenticated.
+
+### 12. Displays logged in profile's WebID when available
```

Comment:
Is this without using the extension, like test 11?

Reply:
in this test case, the WebID was entered in the app input field itself, so without the extension
but it doesn't really matter how the authentication flow was initiated, it should work either way

Reply:
But the extension is stilled installed?

Reply:
The description in the docs doesn't specify whether the extension is installed or not, because it doesn't affect the test case. It's testing something that is not related to the extension.
It's possible to create test case for all possible permutations regarding installed/not-installed extension, but I don't see the need for it.
What do you think? Would you prefer an additional use case without extension installed?

Reply:
I could also add a sentence that indicates whether the extension is installed. Or that indeed it doesn't matter whether it's installed or not.

Reply:
It has to be clear whether it's installed. A test cannot be ambiguous.

Reply:
✔️

Reply:
@pheyvaer
I added info about the extension being installed, and also added a check in the test